client/my-sites/exporter/export-card/advanced-settings.jsx

```diff
@@ -64,7 +64,7 @@ class AdvancedSettings extends React.Component {
 					loading={ this.props.shouldShowProgress }
 					isPrimary={ false }
 					onClick={ this.props.onClickExport }
-					text={ this.props.translate( 'Export Selected Content' ) }
+					text={ this.props.translate( 'Export selected content' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.